wiki/markdown/running-eiffel-intelligence-frontend.md

```diff
@@ -0,0 +1,44 @@
+# Running Eiffel Intelligence Front-end
+
+## _Prerequisites_
+
+Eiffel intelligence Front-end is a Spring microservice distributed in a war file. To start
```

Comment:
Same comments in this file, use lower case and hyphened spelling of back-end and front-end references. Also add a comma after "To start the service"

Reply:
fixed.